specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/common.json

```diff
@@ -60,14 +60,48 @@
           "$ref": "#/definitions/ResourceNotificationsResourceUpdatedDetails",
           "x-ms-client-name": "resourceDetails"
         },
+        "apiVersion": {
```

Comment:
This diff seems wrong. Can you please rebase/pull from main?

Reply:
done!